sopel/tools/time.py

```diff
@@ -132,13 +150,10 @@ def format_time(db=None, config=None, zone=None, nick=None, channel=None,
         tformat = '%Y-%m-%d - %T%Z'
 
     if not time:
-        time = datetime.datetime.utcnow()
+        time = datetime.datetime.now(utc)
 
-    if not pytz or not zone:
-        return time.strftime(tformat)
-    else:
-        if not time.tzinfo:
-            utc = pytz.timezone('UTC')
-            time = utc.localize(time)
+    if pytz and zone:
         zone = pytz.timezone(zone)
-        return time.astimezone(zone).strftime(tformat)
+    if not zone:
+        zone = utc
+    return time.astimezone(zone).strftime(tformat)
```

Comment:
What happens `if zone and not pytz`? I'm not clear on this restructured logic, but I do want to have this feature.